drizzlepac/hlautils/poller_utils.py

```diff
@@ -298,23 +298,34 @@ def determine_filter_name(raw_filter):
       name second (e.g., 'f606w-pol60').
     - NOTE: There should always be at least one filter name provided to
       this routine or this input is invalid.
+
+    - If one filter is 'n/a', then convert to 'none' (as the slash causes problems
+      on Linux-like systems) and always put this filter name second (e.g., 'pr130l-none').
+    - NOTE: At this time (February 2020) the filter names for the SBC detector of
+      ACS have malformed names (e.g., F140LP;N/A;F140LP) where the final token delimited by
+      the ";" should not be present. Remove the final delimiter and entry.
     """
 
     raw_filter = raw_filter.lower()
 
-    # There might be two filters, so split the filter names into a list
-    filter_list = raw_filter.split(';')
+    # There might be multiple filters, so split the filter names into a list
+    # and only retain the first two entries
+    filter_list = raw_filter.split(';')[0:2]
     output_filter_list = []
 
     for filt in filter_list:
+        if filt == 'n/a':
+            filt = 'none'
         # Get the names of the non-clear filters
         if 'clear' not in filt:
             output_filter_list.append(filt)
 
     if not output_filter_list:
         filter_name = 'clear'
+    elif output_filter_list[0].startswith('pol'):
+        output_filter_list.reverse()
     else:
-        if output_filter_list[0].startswith('pol'):
+        if output_filter_list[0].startswith('none'):
```

Comment:
In [1009]: rawfilt = 'POL120V;F555W'                                                                                                

In [1010]: determine_filter_name(rawfilt)                                                                                           
---------------------------------------------------------------------------
UnboundLocalError                         Traceback (most recent call last)
<ipython-input-1010-230fd5bd86d9> in <module>
----> 1 determine_filter_name(rawfilt)

<ipython-input-998-a241b87a1552> in determine_filter_name(raw_filter)
     25         filter_name = delimiter.join(output_filter_list)
     26 
---> 27     return filter_name

UnboundLocalError: local variable 'filter_name' referenced before assignment